include/libxnvme_znd.h

```diff
@@ -218,18 +215,17 @@ XNVME_STATIC_ASSERT(sizeof(struct xnvme_znd_report) == 64, "Incorrect size")
 
 /**
  * Access Zone Descriptors, by entry index, in the given znd_report
+ *
+ * NOTE: the MACRO is kept for backwards compatibility with the upstream fio
+ * io-engine. It will be most likely be deprecated in the future.
  */
+int
+xnvme_znd_report_get_descr(const struct xnvme_znd_report *report, uint32_t index,
+			   struct xnvme_spec_znd_descr **descr, void **ext);
```

Comment:
Do `descr` and `ext` have to be double pointers?

Reply:
They are there, such that the function can checker whether the caller wants to retrieve the descriptor and/or the extended descriptor. The double-pointer indicates that when the value points to a pointer, the caller is requesting that type of information.